### DISCUSSION TITLE
Saving Images?

### DISCUSSION THREAD

**Discussion by ProtoBelisarius** [kaangiray26/geddit-app#67] [Q&A]
How do I go about saving images?
Is that not yet implemented or what do I need to do for it to work?
I just installed the newest 1.7 apk from github but there is neither a button nor does swiping/pressing/tapping the image start the download. 
File access permission was granted.
Thanks

**Comment by saltyollpheist** [comment]
Wondering the same. I've tried delving into the limited settings menu, everything mentioned by OP, and still nothing seems to offer any sort of prompt to save an image anywhere. Gallery is essentially useless. Any input on this by the developer would be appreciated as it's been 2 weeks since this question was originally posted. Otherwise thanks for an app to view Reddit all the same in some form and fashion.

**Comment by kaangiray26** [comment]
Hi everyone,
I'm well aware of this issue and I think I've fixed it for the upcoming version, you can check it out in the commits for v1.8, there is an issue for the upcoming features.

However, there is a problem I've been dealing with. I've received a cease-and-desist letter from Reddit about me making a commercial product via bypassing API restrictions, which is not the case and never will be. This project is and will always be non-commercial, free and open-source. I've been trying to deal with this problem silently but I haven't received any response from Reddit yet, so until further notice from them I'm currently taking a break to work on other things.

**Reply by saltyollpheist** [reply]
Typical corpo Reddit behavior. Thank you for the update and your diligence, kaangiray. If for nothing else but your sanity and financial well-being I hope that gets resolved swiftly. Have a terrific weekend.

**Reply by brantcy** [reply]
Bummer about their shitty letter.
